src/types/system.ts

```diff
@@ -0,0 +1,22 @@
+import { PropChangedAction } from 'plugins/props-change-watcher/actions';
+import { Action } from 'types/actions';
+
+export interface SystemValues {
+  getComponent: () => void;
```

Comment:
We could provide a type here as well, I think it would be like this:
```suggestion
  getComponent: (componentName: string, container?: boolean, config?: object) => ComponentType | null;
```